apm-sniffer/apm-agent-core/src/main/java/org/skywalking/apm/agent/core/plugin/PluginCfg.java

```diff
@@ -12,15 +13,29 @@
 public enum PluginCfg {
     INSTANCE;
 
-    private List<String> pluginClassList = new ArrayList<String>();
+    private List<PluginDefine> pluginClassList = new ArrayList<PluginDefine>();
+    protected List<String> disabledPlugins = new ArrayList<String>();
```

Comment:
DisableList is only used in plugin initialization, shouldn't set it as a class field.